requirements.txt

```diff
@@ -24,7 +24,7 @@ lazy-object-proxy==1.4.3
 mccabe==0.6.1
 networkx==2.5
 numpy==1.19.4
-oauth2client==4.1.3
+oauth2client<4.0.0
```

Comment:
Was running into errors relating to `file_cache` when using `oauth2client` v4.0.0 or later.